jobserver/emails.py

```diff
@@ -25,3 +25,22 @@ def send_finished_notification(email, job):
         template_name="emails/notify_finished.txt",
         context=context,
     )
+
+
+def send_project_invite_email(email, project, invite):
+    f = furl(settings.BASE_URL)
+    f.path = project.get_absolute_url()
+
+    context = {
+        "inviter_name": invite.created_by.name,
+        "project_name": project.name,
+        "url": f.url,
+    }
+
+    send(
+        to=email,
+        sender="notifications@jobs.opensafely.org",
+        subject=f"[os] Project Invite from {invite.created_by.name} to {project.name}",
```

Comment:
Will an invited user know what [os] is?  Maybe change to [OpenSAFELY]?

Reply:
Oh, hmm, this was copying the notification email subject prefix… but now I think about it, that's not really the same as an invite.